float32                             desired_height
float32                             x_size
float32                             y_size
float32                             x_offset
float32                             y_offset
float32                             x_takeOff
float32                             y_takeOff
---
bool                                success
